addOns/retire/src/test/java/org/zaproxy/addon/retire/UpstreamJsRepositoryUnitTest.java

```diff
@@ -31,7 +38,29 @@ class UpstreamJsRepositoryUnitTest {
     void shouldParseUpstreamJsRepository() {
         // Given
         String path = "/org/zaproxy/addon/retire/resources/jsrepository.json";
-        // When / Then
-        assertDoesNotThrow(() -> new Repo(path));
```

Comment:
This assertion should be kept and used the returned repo.

Reply:
How can I get the returned repo out of the lambda?

Reply:
The assert returns it, just needs to assign to a variable.

Reply:
Oh cool!